### ISSUE TITLE
Unable to Send Message on Chrome 54.x

### ISSUE THREAD

**Issue by toreyhickman** [common-nighthawk/hello-its-me#44]
Hey Daniel,
I just sent you a couple messages from the application.  In order for the messages to send, I had to update Chrome (55.0.2883.95) and Firefox to their latest versions.

Prior to the update, I had unsuccessfully sent a message using Chrome 54.x (full version number in one of the messages).  Just documenting the issue in case you wanted to support older browsers.

---

@common-nighthawk

**Comment by common-nighthawk** [comment]
you're amazing!  thank you so much, @toreyhickman!

ill look into why theres an error with 54.0.240.98 x64-bit.  its a bummer, because the documentation says the feature should exist starting with 53.0: https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia

one might say browser compatilibity is no joke: http://danielallendeutsch.com/blog/17-browser-compatibility.html

sorry my fun messaging service is giving you so many errors.  but i really really appreciate all the detailed reports.

(off topic--i went on an awesome hike today: https://goo.gl/photos/PH779fBtCFSn89277)

**Comment by toreyhickman** [comment]
Yeah, I was surprised that I was unable to send a message with the older version of Chrome.  Both because I'd sent one with it before and really it's only a couple months old.  Oh well.

The landscape out there looks awesome!
